pkg/cmd/cli/schedule/create.go

```diff
@@ -94,6 +95,7 @@ func NewCreateOptions() *CreateOptions {
 func (o *CreateOptions) BindFlags(flags *pflag.FlagSet) {
 	o.BackupOptions.BindFlags(flags)
 	flags.StringVar(&o.Schedule, "schedule", o.Schedule, "a cron expression specifying a recurring schedule for this backup to run")
+	flags.BoolVar(&o.UseOwnerReferencesInBackup, "use-owner-rferences-in-backup", o.UseOwnerReferencesInBackup, "specifies whether to use OwnerReferences on backups created by this Schedule")
```

Comment:
There is a typo in this flag, it should be "use-owner-r**e**ferences-in-backup".

Reply:
yes, sorry for my mistake.